.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,10 @@
+contact_links:
+  - name: Forum
+    url: https://forum.nim-lang.org
+    about: Please ask and answer questions here.
+  - name: Discord
+    url: https://discord.com/invite/nim
+    about: Please ask and answer questions here.
```

Comment:
```suggestion
```

Reply:
actually, I think it's good to mention discord. User may be missing out on what happens there.